internal/bminventory/inventory.go

```diff
@@ -466,10 +466,7 @@ func (b *bareMetalInventory) getNewClusterReleaseImage(ctx context.Context, para
 		return nil, errors.Wrapf(err, "Openshift version %s for CPU architecture %s is not supported",
 			swag.StringValue(params.OpenshiftVersion), arch)
 	}
-	if models.OpenshiftVersionSupportLevelMaintenance == releaseImage.SupportLevel {
-		return nil, errors.Errorf("Openshift version %s support level is: %s, and can't be used for creating a new cluster",
-			swag.StringValue(params.OpenshiftVersion), releaseImage.SupportLevel)
-	}
+
```

Comment:
We're no longer handling 'maintenance' support level?

Reply:
We do, but for some reason there was a limitation in the service code that prohibited using maintenance release images for installing cluster, I removed it because we want to support this option.